code/client/repoclean

```diff
@@ -439,6 +444,9 @@ def main():
     parser.add_option('--plugin', default=pref('plugin'),
                       help='Optional plugin to connect to repo. If specified, '
                            'overrides any plugin specified via --configure.')
+    parser.add_option('--auto', '-a', action='store_true',
+                      help='Automatically respond "y" to an y/n questions, '
```

Comment:
typo!

Reply:
Since that text needs to be fixed anyway, might I suggest "Do not prompt for confirmation before deleting repo items. Use with caution."